tests/test_core.py

```diff
@@ -456,6 +456,7 @@ def test_normalize_query():
     [
         (2, False),
         ('foo', False),
+        ('foo\\*bar', False),
```

Comment:
What about a single `\*`?  Doesn't that properly escape the `*`?

Reply:
Good catch! I overlooked this. This is now fixed.

Reply:
...See my comment below.  Apparently, in Python, `'\\*' == '\*'`.  So, I think you did it correctly.